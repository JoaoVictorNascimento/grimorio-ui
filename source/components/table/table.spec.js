import '../../../internals/test/helper';

import Table from './table-component';

/** @test {Table} */
describe('Table component', () => {
/** @test {Table#render} */
  describe('#render', () => {
    it('render correctly', () => {
      const wrapper = shallow(
        <Table />
      );
      expect(wrapper.length).toEqual(1);
    });
  });
});
