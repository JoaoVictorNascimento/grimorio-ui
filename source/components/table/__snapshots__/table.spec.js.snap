// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table component #render with data when renders correctly 1`] = `
"<div style={{...}} className=\\"wrap\\">
  <table className=\\"table\\">
    <thead className=\\"tableHead\\">
      <tr style={{...}} className=\\"rowHead\\">
        <th width=\\"100px\\" className=\\"cellHead\\">
          Nome
        </th>
        <th width={[undefined]} className=\\"cellHead\\">
          <Button onClick={[Function: onClick]} disabled={false} color=\\"primary\\" size=\\"medium\\" type=\\"button\\" className={[undefined]} active={false}>
            Diga oi para todos!
          </Button>
        </th>
      </tr>
    </thead>
    <tbody className=\\"tableBody\\">
      <tr style={{...}} className=\\"row\\">
        <td width=\\"100px\\" className=\\"cell\\">
          Teste
        </td>
        <td width={[undefined]} className=\\"cell\\">
          <Button onClick={[Function: onClick]} disabled={false} color=\\"primary\\" size=\\"medium\\" type=\\"button\\" className={[undefined]} active={false}>
            Adivinhe minha idade ;)
          </Button>
        </td>
      </tr>
    </tbody>
  </table>
</div>"
`;
